makelib/golang.mk

```diff
@@ -171,7 +173,7 @@ ifeq ($(GO_NOCOV),true)
 else
 	@mkdir -p $(GO_TEST_OUTPUT)
 	@CGO_ENABLED=0 $(GOHOST) test -i -cover $(GO_STATIC_FLAGS) $(GO_PACKAGES) || $(FAIL)
```

Comment:
is it odd that this `test` command will always run with `CGO_ENABLED=0`, but the `test` command on the next line will use the `CGO_ENABLED` variable?

Reply:
Sure, I could change this line. But I noticed the `@CGO_ENABLED=0` was hard coded lots of places, I don't want this PR make widely changes